orcid-core/src/main/resources/i18n/api_en.properties

```diff
@@ -61,7 +61,7 @@ apiError.9018.userMessage=The ORCID record is locked.
 apiError.9018.moreInfo=http://members.orcid.org/api/api-error-codes
 apiError.9019.userMessage=The put code in the URL did not match that in the body of the message.
 apiError.9019.moreInfo=http://members.orcid.org/api/api-error-codes
-apiError.9020.userMessage=The put code in the URL did not match that in the body of the message.
+apiError.9020.userMessage=Invalid incoming message.
```

Comment:
(We do also include the exception message in developerMessage in the error response, though of course that will be in English. The userMessage is for the client to display to the user, so I suspect we might not want too much info in there unless it is something the user can fix themselves. I also think it might be helpful to our support team if the message said something to indicate that the user should complain to the client application support to team, not to ORCID!)